schema_based_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE patients(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(250),
+  date_of_birth DATE,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  admitted_at timestamp,
+  patient_id INT,
+  FOREIGN KEY (patient_id) REFERENCES patients(id),
+  status VARCHAR(250),
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE invoices(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  total_amount decimal,
+  generated_at timestamp,
+  payed_at timestamp,
+  medical_history_id INT,
+  FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE treatments(
+id INT GENERATED ALWAYS AS IDENTITY,
+type VARCHAR(250),
+name VARCHAR(250),
+PRIMARY KEY(id)
+);
+
+CREATE TABLE invoice_items(
+id INT GENERATED ALWAYS AS IDENTITY,
+unit_price decimal,
+quantity INT,
+total_price decimal,
+invoice_id INT,
+treatment_id INT,
+FOREIGN KEY (invoice_id) REFERENCES invoices(id),
+FOREIGN KEY (treatment_id) REFERENCES treatments(id),
+PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories_treatments(
+id INT GENERATED ALWAYS AS IDENTITY,
+medical_histories_id INT,
+treatments_id INT,
+FOREIGN KEY (medical_histories_id) REFERENCES medical_histories(id),
+FOREIGN KEY (treatments_id) REFERENCES treatments(id),
+PRIMARY KEY(id)
+); 
```

Comment:
- [ ] @limavfabio and @IngDavidTM great work on your database creation,however, you forgot to add foreign keys indexes which is part of the requirements

![image](https://user-images.githubusercontent.com/61408860/207429014-03013205-e918-4acc-85b8-f5d7366c272d.png)

Reply:
well resolved! ✔️